hq/app/aws/support/TrustedAdvisor.scala

```diff
@@ -54,6 +57,36 @@ object TrustedAdvisor {
     handleAWSErrs(awsToScala(client.describeTrustedAdvisorCheckResultAsync)(request))
   }
 
+  private[support] def sortSecurityFlags[A <: TrustedAdvisorCheckDetails](list: List[A]): List[A] = {
+    //    AWS Alert mappings
+    //    Green: Access to port 80, 25, 443, or 465 is unrestricted.
+    //    Red: Access to port 20, 21, 1433, 1434, 3306, 3389, 4333, 5432, or 5500 is unrestricted.
+    //    Yellow: Access to any other port is unrestricted.
+    //
+    //    FTP : 21
```

Comment:
I think this documentation should be with the port definitions, not here?

Reply:
I guess I can add that info to ports and propagate it back to the UI for display